<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>857</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" alignment="Qt::AlignVCenter">
     <widget class="QTabWidget" name="find">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Edit</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_12">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFontComboBox" name="fontComboBox">
            <property name="editable">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QPushButton" name="backcolor">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="backgroundCombo">
                <item>
                 <property name="text">
                  <string>White</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Black</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Gray</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Red</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Orange red</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Gold</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Green yellow</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Lawn green</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Light green</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Pale green</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Aqua</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Dodger Blue</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Blue</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Royal Blue</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Orchid</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Fuchsia</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Deep pink</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="Line" name="line_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="Line" name="line_4">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QPushButton" name="forecolor">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="foregroundCombo">
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <item>
                 <property name="text">
                  <string>Black</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Gray</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Red</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Orange red</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Gold</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Green yellow</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Lawn green</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Light green</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Pale green</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Aqua</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Dodger Blue</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Blue</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Royal Blue</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Orchid</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Fuchsia</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Deep pink</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>White</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,0,0,0">
            <property name="spacing">
             <number>20</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="alignLeft">
              <property name="minimumSize">
               <size>
                <width>30</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>30</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="myres.qrc">
                <normaloff>:/Editing-Align-Left-icon.png</normaloff>:/Editing-Align-Left-icon.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="alignCenter">
              <property name="minimumSize">
               <size>
                <width>30</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>30</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="myres.qrc">
                <normaloff>:/Editing-Align-Center-icon.png</normaloff>:/Editing-Align-Center-icon.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="alignRight">
              <property name="minimumSize">
               <size>
                <width>30</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>30</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="myres.qrc">
                <normaloff>:/Editing-Align-Right-icon.png</normaloff>:/Editing-Align-Right-icon.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="justify">
              <property name="minimumSize">
               <size>
                <width>30</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>30</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="myres.qrc">
                <normaloff>:/Editing-Align-Justify-icon.png</normaloff>:/Editing-Align-Justify-icon.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <property name="sizeConstraint">
             <enum>QLayout::SetFixedSize</enum>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QPushButton" name="underline">
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="myres.qrc">
                  <normaloff>:/Editing-Underline-icon.png</normaloff>:/Editing-Underline-icon.png</iconset>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="italic">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="sizeIncrement">
                 <size>
                  <width>35</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="baseSize">
                 <size>
                  <width>1</width>
                  <height>1</height>
                 </size>
                </property>
                <property name="focusPolicy">
                 <enum>Qt::WheelFocus</enum>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="myres.qrc">
                  <normaloff>:/Editing-Italic-icon.png</normaloff>:/Editing-Italic-icon.png</iconset>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="bold">
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="myres.qrc">
                  <normaloff>:/Editing-Bold-icon.png</normaloff>:/Editing-Bold-icon.png</iconset>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="sizebox">
                <item>
                 <property name="text">
                  <string>8</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>9</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>10</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>11</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>12</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>14</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>16</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>18</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>20</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>22</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>24</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>26</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>28</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>36</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>48</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>72</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>60</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Find</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLineEdit" name="findQuery"/>
          </item>
          <item>
           <widget class="QPushButton" name="findButton">
            <property name="text">
             <string>Find Next</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="findPreviousButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>PushButton</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QTextEdit" name="textEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>857</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionLoad"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionREdo"/>
    <addaction name="separator"/>
    <addaction name="actionCopy"/>
    <addaction name="actionCut"/>
    <addaction name="actionPaste_2"/>
    <addaction name="actionSelectAll"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionREdo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionPaste_2">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="text">
    <string>SelectAll</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="myres.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>test()</slot>
 </slots>
</ui>
